src/Yoti/Util/Profile/AttributeConverter.php

```diff
@@ -166,6 +166,12 @@ public static function convertToYotiAttribute(ProtobufAttribute $protobufAttribu
     {
         $yotiAttribute = null;
 
+        // Application Logo can be empty, return NULL when this occurs.
+        if ($protobufAttribute->getName() == Profile::ATTR_APPLICATION_LOGO &&
```

Comment:
We have separate constants for UserProfile and ApplicationProfile, you can find the application_logo const [here](https://github.com/getyoti/yoti-php-sdk/blob/master/src/Yoti/Entity/ApplicationProfile.php#L7).

Reply:
Good spot 👍 fixed in 3655638